nixos/release.nix

```diff
@@ -92,15 +92,33 @@ let
   }).config));
 
 
+  cleanConfig = import ./lib/eval-config.nix {
```

Comment:
Aha, so it would be best to also make the local manual generation use this way, right?